packages/react-core/src/components/Wizard/WizardDrawerWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export interface DrawerWrapperProps {
+  children: React.ReactElement;
+  condition: boolean;
```

Comment:
I would still add prop description her for other devs maintaining the code.
I would also make the condition name more descriptive o the condition that is being checked. to

Reply:
Great idea/suggestions - added prop descriptions here and changed `condition` to `hasDrawer`!